### PR TITLE
Tyr: handle raw files or zips

### PR DESCRIPTION
**Pull request by antoine-de**
For artemis not to need zips, tyr has to be changed.

Tyr can now handle either zips or directory

DO NOT MERGE IT YET, i've tested it only with artemis (and it works), but not with a real 'prod' env, and it not anymore the same thing

### CODE REVIEW

File: source/tyr/tyr/binarisation.py [hunk 1]
```diff
@@ -127,10 +127,13 @@ def fusio2ed(self, instance_config, filename, job_id, dataset_uid):
 
     logger = get_instance_logger(instance)
     try:
-        working_directory = os.path.dirname(filename)
-
-        zip_file = zipfile.ZipFile(filename)
-        zip_file.extractall(path=working_directory)
+        if not os.path.isdir(filename):
```
Comment (pbougue) [inline source/tyr/tyr/binarisation.py]: It's worth a utils function, no?

File: source/tyr/tyr/binarisation.py [hunk 2]
```diff
@@ -127,10 +127,13 @@ def fusio2ed(self, instance_config, filename, job_id, dataset_uid):
...
+            # if it's not a directory, we consider it's a zip, and we unzip it
```
Comment (pbougue) [inline source/tyr/tyr/binarisation.py]: Could we test if it ends by ".zip", or if it unzips correctly? We may allow a single file, like poi?

File: source/tyr/tyr/binarisation.py [hunk 3]
```diff
@@ -127,10 +127,13 @@ def fusio2ed(self, instance_config, filename, job_id, dataset_uid):
...
+            working_directory = os.path.dirname(filename)
+            zip_file = zipfile.ZipFile(filename)
+            zip_file.extractall(path=working_directory)
+        else:
```
Comment (pbougue) [inline source/tyr/tyr/binarisation.py]: not really up to date on tyr, but what if we get a dir that contains only a zip file?

**Comment by pbougue** [comment]
Is it possible to test with eitri/docker_test?

File: source/tyr/tyr/binarisation.py (hunk 3 above)
Comment (xlqian) [inline source/tyr/tyr/binarisation.py]: @pbougue hmm, I think that test by trying unzip is simple enough

File: source/tyr/tyr/binarisation.py [hunk 4]
```diff
@@ -166,10 +169,12 @@ def gtfs2ed(self, instance_config, gtfs_filename, job_id, dataset_uid):
 
     logger = get_instance_logger(instance)
     try:
-        working_directory = os.path.dirname(gtfs_filename)
-
-        zip_file = zipfile.ZipFile(gtfs_filename)
-        zip_file.extractall(path=working_directory)
+        if not os.path.isdir(gtfs_filename):
```
Comment (xlqian) [inline source/tyr/tyr/binarisation.py]: it looks like repeated codes

File: source/tyr/tyr/binarisation.py (hunk 2 above)
Comment (antoine-de) [inline source/tyr/tyr/binarisation.py]: nop because not all ziped files end with .zip

**Comment by antoine-de** [comment]
nop test tyr is a task (and not a short one) on its own.

**Comment by pbougue** [comment]
Ok for me.

**Comment by pbougue** [comment]
Tested by @antoine-de so I merge

**Pull request merged**